scipio/src/file_stream.rs

```diff
@@ -667,7 +667,7 @@ impl StreamWriterBuilder {
     ///
     /// [`StreamWriter`]: struct.StreamWriter.html
     pub fn with_write_behind(mut self, write_behind: usize) -> Self {
-        self.write_behind = write_behind;
+        self.write_behind = std::cmp::max(write_behind, 1);
```

Comment:
This is a good idea. We should advertise than a value of `0` means "left to the system to decide" in the documentation.

Reply:
I don't think `0` means left the system to decide. Not calling this means left to the system to decide.

What we are doing here is just enforcing sane boundaries.